.github/workflows/ci_mac.yml

```
@@ -199,6 +199,19 @@ jobs:
           exit $rc
         fi
 
+    - name: Testing GUI
+      if: matrix.skip_testing != true
+      working-directory: tests
+      run: |
+        export ASAN_OPTIONS=fast_unwind_on_malloc=0
+        # Exclude tests that are currently known to fail
+        export wx_tests_selection='~[wxTextCtrl] ~[item-container] ~[persist] ~[html] ~[font] ~[wxWebView] ~[valnum] ~[radiobutton] ~[clip] ~wxExecute::RedirectUTF8 ~Ellipsization::NormalCase ~SpinCtrl::Wrap ~SpinCtrlDouble::Arrows ~SpinCtrlDouble::Wrap ~SpinCtrl::Arrows ~SpinCtrlDouble::Increment ~Grid::KeyboardSelection ~NotebookTestCase ~TextCtrlTestCase ~SettingsTestCase ~XRC::LoadURL'
+        # Explicitly use bash because /bin/sh doesn't have pipefail option
+        /bin/bash -o pipefail -c './test_gui $wx_tests_selection 2>&1 | tee test_gui.out' || rc=$?
```

Shouldn't this be
```suggestion
        /bin/bash -o pipefail -c "./test_gui $wx_tests_selection 2>&1 | tee test_gui.out" || rc=$?
```
to expand the variables?